src/config/urls.config.ts

```diff
@@ -0,0 +1,29 @@
+export interface PagedQueryInterface {
+    page: number;
+    limit: number;
+    [key: string]: string | number;
+}
+
+export interface PostsPagedQueryInterface extends PagedQueryInterface {
+    userId: number;
+}
+
+export interface TodosPagesQueryInterface extends PagedQueryInterface {
+    userId: number;
+}
+
+const getQueryString = <T extends Record<string, number | string>>(obj: T): string => {
+    const uri = new URLSearchParams();
+    for (const key in obj) {
+        uri.append(encodeURIComponent(key), encodeURIComponent(obj[key]));
+    }
+    return uri.toString();
+}
+
+export const baseUrl = "http://localhost:3001";
```

Comment:
Use CRA env variables instead `process.env.REACT_APP_BASE_URL`
And then the `.env.development` file will have 
```
REACT_APP_BASE_URL=http://localhost:3001
```